src/Feed/BrndFeedType.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Feed;
+
+use App\Entity\Tenant\Feed;
+use App\Entity\Tenant\FeedSource;
+use App\Feed\SourceType\Brnd\ApiClient;
+use App\Feed\SourceType\Brnd\SecretsDTO;
+use App\Service\FeedService;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Brnd Bookingsystem Feed.
+ *
+ * @see https://brndapi.brnd.com/swagger/index.html
+ */
+class BrndFeedType implements FeedTypeInterface
+{
+    public const int CACHE_TTL = 3600;
+
+    final public const string SUPPORTED_FEED_TYPE = FeedOutputModels::BRND_BOOKING_OUTPUT;
+
+    public function __construct(
+        private readonly FeedService $feedService,
+        private readonly ApiClient $apiClient,
+        private readonly CacheItemPoolInterface $feedsCache,
+    ) {}
+
+    public function getAdminFormOptions(FeedSource $feedSource): array
+    {
+        $feedEntryRecipients = $this->feedService->getFeedSourceConfigUrl($feedSource, 'sport-center');
+
+        return [
+            [
+                'key' => 'brnd-sport-center-id',
+                'input' => 'input',
+                'type' => 'text',
+                'name' => 'sport_center_id',
+                'label' => 'Sportcenter ID',
+                'formGroupClasses' => 'mb-3',
+            ],
+        ];
+    }
+
+    public function getData(Feed $feed): array
+    {
+        $result = [
+            'title' => 'BRND Booking',
+            'bookings' => [],
+        ];
+
+        $configuration = $feed->getConfiguration();
+        $feedSource = $feed->getFeedSource();
+
+        if (null == $feedSource) {
+            return $result;
+        }
+
+        $secrets = new SecretsDTO($feedSource);
+
+        $baseUri = $secrets->apiBaseUri;
+        $sportCenterId = $configuration['sport_center_id'] ?? null;
+
+        if ('' === $baseUri || null === $sportCenterId || '' === $sportCenterId) {
+            return $result;
+        }
+
+        $feedSource = $feed->getFeedSource();
+
+        if (null === $feedSource) {
+            return $result;
+        }
+
+        $bookings = $this->apiClient->getInfomonitorBookingsDetails($feedSource, $sportCenterId);
+
+        $result['bookings'] = array_map([$this, 'parseBrndBooking'], $bookings);
+
+        return $result;
+    }
+
+    private function parseBrndBooking(array $booking): array
+    {
+        // Parse start time
+        $startDateTime = null;
+        if (!empty($booking['dato']) && isset($booking['starttid']) && is_string($booking['starttid'])) {
+            // Trim starttid to 6 digits after dot for microseconds
+            $starttid = preg_replace('/\.(\d{6})\d+$/', '.$1', $booking['starttid']);
```

Comment:
Replace danish variable name with english.
Maybe: $startTimeString

Reply:
The danish varible names are in the response object returned from the BRND webservice. They have choosen to code their webservice in danish. An odd choice, but I can not change that. I have attached an example of a webservice response so you can see it for yourself.

Reply:
[response_1761723886623.json](https://github.com/user-attachments/files/23233436/response_1761723886623.json)